### PR TITLE
Update video reading process

### PR DESCRIPTION
**Pull request by inuzukam**
## What?
- Update video reading process

## Why?
- To reduce the number of frames to read and improve efficiency

### CODE REVIEW

File: pydtk/models/movie.py [hunk 1]
```diff
@@ -39,47 +39,55 @@ def _load(self, path, start_timestamp=None, end_timestamp=None, **kwargs):
...
+        if end_timestamp is not None:
+            end_frame_idx = min(np.ceil(end_timestamp * fps).astype(int), n_frames - 1)
+        else:
+            end_frame_idx = n_frames - 1
+        frame_size = end_frame_idx - start_frame_idx + 1
+        assert 0 <= start_frame_idx < end_frame_idx < n_frames, "Timestamp out of range!"
+
+        # Read video
+        cap.set(cv2.CAP_PROP_POS_FRAMES, start_frame_idx)
+        data = np.empty((frame_size, height, width, n_channels))
```
Comment (d-hayashi) [inline pydtk/models/movie.py:69]: `data` should have dtype of unsigned int8 ([default is float](https://numpy.org/doc/stable/reference/generated/numpy.empty.html)).

**Comment by d-hayashi** [comment]
@inuzukam 
Thank you for your contribution!  
Could you check the comment so that all tests pass.

File: pydtk/models/movie.py (hunk 1 above)
Comment (inuzukam) [reply pydtk/models/movie.py]: Fixed the code!

**Review by d-hayashi — APPROVED** [review]
LGTM!

**Review by kan-bayashi — APPROVED** [review]
LGTM
Did you check the loading speed difference?

**Comment by inuzukam** [comment]
@kan-bayashi 
In my one sample, by specifying start_timestamp and end_timestamp, the reading video that used to take about 1 hour now takes about 2 minute!

**Comment by kan-bayashi** [comment]
Really COOL!

**Pull request merged**